shared/actions/deeplinks.tsx

```diff
@@ -93,7 +94,7 @@ const handleAppLink = (state: Container.TypedState, action: DeeplinksGen.LinkPay
       }
       return [
         RouteTreeGen.createSwitchTab({tab: Tabs.settingsTab}),
-        RouteTreeGen.createNavigateAppend({path: ['settingsAddPhone']}),
+        SettingsGen.createOpenAddPhoneNumberModal(),
```

Comment:
`createOpenAddPhoneNumberModal` doesn't even exist in settingsgen? or is my editor having trobules?

Reply:
crap i forgot about this flow